docs: Document `--terraform-plan-flags` usage limitations for infracost (#364)
antonbabenko/pre-commit-terraform: README.md

    <!-- markdownlint-disable-next-line no-inline-html -->
    </details>

> 2. Note, that there can be a maximum of one `--terraform-plan-flags` argument. Example of right usage:

Reviewer: ```suggestion
2. Note that there can be a maximum of one `--terraform-plan-flags` argument inside `--args`. Split into several `--args` to avoid using spaces. Example of right usage:
```

> 2. `--terraform-plan-flags` usage limitations. Please, set only one flag per `--terraform-plan-flags` or hook will crash:

Reviewer: ```suggestion
2. Note, that there can be a maximum of one `--terraform-plan-flags` argument.
```

And write the correct config below. Currently, this confuses more than helps :)

    ```yaml
    - id: infracost_breakdown
      args:
        - --args=--path=./env/dev

>         - --args=--terraform-plan-flags="-var-file=terraform.tfvars"

Reviewer: Well, there are two arguments (lines 308 and 309). As I said, it is confusing, and I don't know what is the right config as a user :)
